Gravatar.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use Nette\Utils\Validators;
+
+class Gravatar
+{
+	private string $email;
+	private string $hash;
+
+	public function getIcon(string $email): string
+	{
+		/*
+		if (!Validators::isEmail($email)) {
+			throw new InvalidArgumentException("invalid email");
```

Comment:
Proč je zakomentováno? Logika je správně.

Reply:
Výjimka se musí prefixovat:

```suggestion
			throw new \InvalidArgumentException('E-mail "' . $mail . '" is not valid.');
```